### PR TITLE
Fixes GraphQL Upgrader: multi-line fields/connections, and helpful message for :create_base_objects task

### PR DESCRIPTION
**Pull request by esjee**
After trying out the initial version of the upgrader in https://github.com/rmosolgo/graphql-ruby/pull/1110 on the H1 codebase, I ran into some surprises. This PR is supposed to highlight those, and hopefully @Willianvdv can use his regex magic to fix some of these :innocent:

### CODE REVIEW

**Comment by Willianvdv** [comment]
Thanks for PRing the failing specs, this is great!

File: spec/graphql/upgrader/member_spec.rb [hunk 1]
```diff
@@ -219,4 +219,50 @@ class UserInterface < BaseInterface
...
+  end
+
+  describe 'multi-line field with property/method' do
+    it 'upgrades without breaking syntax' do
+      old = %{
+        field :is_example_field, types.Boolean,
+          property: :example_field?
+      }
+      new = %{
+        field :is_example_field, Boolean, null: false
```
Comment (esjee) [inline spec/graphql/upgrader/member_spec.rb]: Unsure if this `null: false` is correct

File: lib/graphql/upgrader/member.rb [hunk 2]
```diff
@@ -16,28 +16,52 @@ def upgrade
...
+            /(?<field_type>field|connection|argument) :(?<name>[a-zA-Z_0-9_]*)?, (?<return_type>.*?(?:,|$|\}))(?<remainder>.*)/
 
           if (matches = field_regex.match(field))
             name = matches[:name]
             return_type = matches[:return_type]
             remainder = matches[:remainder]
-            thing = matches[:thing]
             field_type = matches[:field_type]
 
+            binding.pry if name == '_id'
```
Comment (Willianvdv) [inline lib/graphql/upgrader/member.rb]: 👏

File: spec/graphql/upgrader/member_spec.rb [hunk 3]
```diff
@@ -213,10 +213,87 @@ class UserInterface < BaseInterface
...
       }
       assert_equal upgrade(old), new
     end
   end
+
+  describe 'complicated inline definition' do
+    # it 'upgrades without breaking syntax' do
+    #   old = %{
+    #     connection :example_connection, -> { ExampleConnectionType } do
+    #       argument :order_by, (GraphQL::InputObjectType.define do
```
Comment (Willianvdv) [inline spec/graphql/upgrader/member_spec.rb]: @esjee are you fine extracting this spec into a separate PR? It's pretty challenging to build a regex that matches inline types. We can ship this PR and take our time to fix this specific case.

File: spec/graphql/upgrader/member_spec.rb [hunk 4]
```diff
@@ -213,10 +213,87 @@ class UserInterface < BaseInterface
...
+
+      assert_equal upgrade(old), new
+    end
+  end
+
+  # describe 'field with description inside block' do
+  #   it 'moves the description as an argument' do
+  #     old = %{
+  #       field :_id, !types.ID, property: :id do
+  #         description 'The primary key from the database'
```
Comment (Willianvdv) [inline spec/graphql/upgrader/member_spec.rb]: @esjee can we remove this from the PR and make a new one after you found out what's wrong with the description type?

File: spec/graphql/upgrader/member_spec.rb (hunk 3 above)
Comment (esjee) [reply spec/graphql/upgrader/member_spec.rb]: Sure, works for me.

**Comment by Willianvdv** [comment]
@esjee any idea why every build on CI run fails?

**Comment by esjee** [comment]
No -- but it also fails on the base branch. It seems to have been resolved, I'll merge the base branch back in.

**Comment by esjee** [comment]
And now it works! :tada:

**Comment by Willianvdv** [comment]
@rmosolgo this one is good to go!

**Pull request merged**